spec/helpers/guide_helper_spec.rb

```diff
@@ -2,9 +2,9 @@
 
 RSpec.describe GuideHelper, '#guide_community_options_for_select', type: :helper do
   it 'returns an array of options for a select tag in alphabetical order' do
-    second = create(:guide_community, editions: [build(:edition, title: 'Banana', content_owner: nil)])
-    first = create(:guide_community, editions: [build(:edition, title: 'Apple', content_owner: nil)])
-    third = create(:guide_community, editions: [build(:edition, title: 'Cucumber', content_owner: nil)])
+    second = create(:guide_community, :with_published_edition, title: 'Banana')
+    first = create(:guide_community, :with_published_edition, title: 'Apple')
+    third = create(:guide_community, :with_published_edition, title: 'Cucumber')
```

Comment:
It's tested in https://github.com/alphagov/service-manual-publisher/pull/314/files#diff-1e73b14f1ae43ea29a378b4874d36140R344 but it doesn't hurt to replace one of these with a draft.. I'll update it.